apps/docs/pages/guides/functions/debugging.mdx

```diff
@@ -11,82 +11,135 @@ export const meta = {
 
 Logs are provided for each function invocation, locally and in production.
 
-Any uncaught exceptions thrown by a function during the execution will be automatically logged.
+### How to access logs in production
```

Comment:
Should we have a top level heading `How to access logs` and under it have sub headings `hosted` (prefer hosted over production as that's more accurate) and `local`